src/controllers/routers/staffs.js

```diff
@@ -0,0 +1,40 @@
+/*!
+ * OneSG API Server by TL Bootcamp#6 OneSG Team
+ * Copyright(c) 2020 TechLadies
+ * MIT Licensed
+ */
+
+'use strict';
+
+const url = require('url');
+const querystring = require('querystring');
+
+const { Staff } = require('../../models');
+
+/**
+ * Retrieve all referees
+ * @param {Request} req
+ * @param {Response} res
+ */
+const getAll = async (req, res) => {
+  // to obtain the full url
+  const fullUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
+  // to parse the full url to retrieve all query params
+  const parsedUrl = url.parse(fullUrl);
+  // to break down into individual query params
+  const parsedQueries = querystring.parse(parsedUrl.query);
+  if (parsedQueries.status === 'active') {
+    const results = await Staff.query().where('status', 'ACTIVE');
+    return res.status(200).json({ results });
+  }
+  if (parsedQueries.status === 'disabled') {
+    const results = await Staff.query().where('status', 'DISABLED');
+    return res.status(200).json({ results });
+  }
+  const results = await Staff.query();
```

Comment:
what happens here is if "active" or "disabled" are not matched, then all staff record are returned
since we don't ensure that status is limited to active and disabled
I think we should have that check

Reply:
included a check to check if !== disabled or !== active, throw an error 
return next(new InvalidInput(`Status can only be 'active' or 'disabled', ${parsedQueries.status} is invalid, `));